werkomgeving/gegeven/index.md

```diff
@@ -20,29 +20,29 @@ In bovenstaande voorbeelden wordt met [Jan] de verwijzing bedoeld naar een speci
 
 ## Een gegevenstype
 
-Een gegevenstype is *een typering van gelijksoortige gegevens*. Met een gegevenstype beschrijven we hoe een afzonderlijk gegeven er uit ziet, waarbij bovenstaande vier elementen terugkomen. Van een gegevenstype wil je de volgende zaken weten:
```

Comment:
> Een gegevenstype is *een typering van gelijksoortige gegevens*.

Ook dit voelt voor mij nog niet helemaal correct. Gelijksoortige gegevens kunnen nl. ook ‘kleuren’, ‘vormen’ en ‘geuren’ zijn maar dat is niet wat we hier bedoelen. Er moet iets van abstractie in deze definitie terecht komen. Bijv. iets als ‘een typering van gelijksoortige geabstraheerde gegevens’

Reply:
Eens dit moet scherper gedefinieerd worden.

Reply:
> Ook dit voelt voor mij nog niet helemaal correct. Gelijksoortige gegevens kunnen nl. ook ‘kleuren’, ‘vormen’ en ‘geuren’ zijn maar dat is niet wat we hier bedoelen. Er moet iets van abstractie in deze definitie terecht komen. Bijv. iets als ‘een typering van gelijksoortige geabstraheerde gegevens’

Ik begrijp dat het erg lastig is om hier tot de juiste definitie te komen. Laten we gewoon kijken of dat toch nog ergens in dit proces gaat lukken.